sim/firesim-lib/src/main/cc/bridges/blockdev.cc

```diff
@@ -17,6 +17,7 @@
 /* Uncomment to get DEBUG printing
```

Comment:
Remove these comments.

Reply:
We want this flag commented out by default so that it doesn't affect simulation performance. It basically functions like the TOKENVERIFY flag in the NIC bridge.